service-config/src/main/java/com/palantir/config/service/ServiceDiscoveryConfiguration.java

```diff
@@ -136,6 +136,12 @@
         return getServiceConfig(serviceName).readTimeout();
     }
 
+    public final boolean isServiceEnabled(String serviceName) {
```

Comment:
Deal. Please leave javadoc explaining what "enabled means": `/** Returns true iff the configuration for the given service name exists and its list of URIs is non-empty. */`